manifests/charts/istiod-remote/files/gen-istiod-remote.yaml

```diff
@@ -682,3 +682,9 @@ webhooks:
     namespaceSelector:
       matchLabels:
         istio-injection: enabled
+    objectSelector:
+      matchExpressions:
+      - key: "sidecar.istio.io/inject"
```

Comment:
@istio/wg-environments-maintainers @istio/wg-user-experience-maintainers  we have an interesting case here... we have always had `sidecar.istio.io/inject` be documented/used as an annotation that is filtered in the injector code.

Filtering it in the webhook config is, as far as I can tell, strictly better.

Should we start moving people to use the label instead? Of course the annotation will be around for backwards compat probably forever.

Reply:
As an outsider, I say we should have people move to the label. 

Right now, we set both the label and the annotation

Reply:
My take is to standardize on labels in the future. We are definitely stuck with the annotation. Many open K8s software treat these two independent fields as having the same meaning.

Reply:
Honestly, as far as K8s is concerned, `Annotations` are the right place for this given what they are meant to do. It's unfortunate that mutating webhooks selects only labels when using `objectSelector`